src/plugins/vis_type_timeline/server/types.ts

```diff
@@ -29,3 +29,17 @@
  */
 
 export { TimelineFunctionInterface, TimelineFunctionConfig } from './lib/classes/timeline_function';
+
+export interface OpenSearchFunctionConfig {
+  q: string | null;
+  metric: string | null;
+  split: string | null;
+  index: string | null;
+  timefield: string | null;
+  kibana: boolean | null;
+  /**
+   * @deprecated The interval picker should be used instead
+   */
+  interval: string | null;
```

Comment:
what is meaning of comment? It is deprecated and we are still using it?

Reply:
+1

Reply:
In the Timeline `expression`, there is an option for the user to specify an interval. However, as noted in the opensearch function `args`, there is a picker react component that users should use instead. I assume interval is kept as to not break any existing visualization.

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/458501f1023df461a2ef904f56b020ee05fc95c3/src/plugins/vis_type_timeline/server/series_functions/opensearch/index.js#L110-L116